src/client/components/pages/error.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2016  Daniel Hsing
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+const React = require('react');
+const Button = require('react-bootstrap').Button;
+
+/**
+ * Links to different pages
+ */
+
+class ErrorPage extends React.Component {
+
+	render() {
+		const {error} = this.props;
+		let detailedMessage = error.detailedMessage;
+		if (typeof detailedMessage === 'string') {
+			detailedMessage = [detailedMessage];
+		}
+
+		return (
+			<div
+				style={{
+					display: 'flex',
+					flexDirection: 'column',
+					alignItems: 'center',
+					justifyContent: 'center'
+				}}
+			>
+				<h1>{error.status}</h1>
+				<b
+					style={{
+						marginTop: '1.5em',
```

Comment:
Hmm, use the margin-top-1 and margin-bottom-1 classes here rather than inline styling (there will be a margin-top-1d5 or equivalent soon, but not added in the master branch yet).